align/affineGap_highMem.go

```diff
@@ -0,0 +1,259 @@
+package align
```

Comment:
Is this a copy of the affineGap.go file from before?  If so, in the future you can do "git mv affineGap.go affineGap_highMem.go" which will do this and it will show up as a simple one-line rename in the diff.  Currently it is difficult to tell what has just been moved to a new file and what has been modified.

Reply:
Yes it is a copy! I did git mv initially but then ran into git merge conflicts after go fmt, and when I resolved those conflicts the affineGap_highMem.go document got uploaded as a new file entirely. I will do that in the future